tests/acceptance/SettingsSubscriptionPageCest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class SettingsSubscriptionPageCest {
+  function previvewDefaultSubscriptionPage(\AcceptanceTester $I) {
+    $I->wantTo('Preview default MailPoet page from MP Settings page');
+    $I->login();
+    $I->amOnMailPoetPage('Settings');
+    $I->click(['xpath'=>'//*[@id="mailpoet_settings_form"]/div[2]/table/tbody/tr[5]/td/p[1]/a']);
```

Comment:
This type of a deeply nested selector tends to be brittle, I would assign an attribute like `data-automation-id="preview-subscription-page"` to this link in the view and then reference it directly from the test. We did this for numerous other acceptance tests, you can check them for reference.